lib/media/ver10/media.ex

```diff
@@ -6,10 +6,9 @@ defmodule Onvif.Media.Ver10.Media do
   """
   require Logger
 
-  @endpoint "/onvif/media_service"
```

Comment:
Since we are removing the hardcoded path in media service API. Should we also remove it from the device service API to be consistent?

Reply:
yes